test/exe/deploy_test.rb

```diff
@@ -79,12 +68,6 @@ def test_deploy_fails_without_filename
     end
   end
 
-  def test_deploy_uses_current_sha
-    test_sha = "TEST"
-    set_krane_deploy_expectations(new_args: { current_sha: test_sha })
-    krane_deploy!(flags: "--current-sha #{test_sha}")
-  end
```

Comment:
Do we need a test to figure out what happens when we pass an ERB template to deploy (e.g. extension `.yml.erb`)?

Reply:
Covered in https://github.com/Shopify/kubernetes-deploy/pull/635/files#diff-ab79bd0231cddd4a971c2848fad61e1cL1647